fs-storage/src/bin/cli.rs

```diff
@@ -0,0 +1,64 @@
+use std::collections::HashMap;
+use std::env;
+use std::path::Path;
+
+use fs_storage::file_storage::FileStorage;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() < 3 {
+        println!("Usage: cargo run -- [read|write] path [key1,key2|key1:value1,key2:value2]");
```

Comment:
This universal message could be confusing during quick evaluation.

I've committed a simpler instruction with separate commands for read and write.